### DISCUSSION TITLE
太记本地数据库的存储方式

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Desktop#590] [Ideas]
https://github.com/TiddlyWiki/TiddlyWiki5/pull/7329

看到这个PR，不知道太记这种nodejs的方式会不会更好一点，刷新速度更快一点。如果会更好的话，那感觉可以做成一个插件来改善。当然可以有两种存储方式，一种SQL的，一种tid的。前者是为了加速访问，后者是方便查看。

我个人觉得太记可以相对tiddlywiki独立一点，改善一些tiddlywiki的不当之处。

**Comment by linonetwo** [comment]
太记移动端已经是 SQLite 的了，不过我发现实际提升不大，因为卡顿主要还是筛选器导致的，包括启动时构建反链和标签 indexer，主要还是得加速这个

**Reply by dongrentianyu** [reply]
那等下新出的mws插件，会有性能上的改进吗？

**Reply by linonetwo** [reply]
应该不会吧，要用别的办法，我在 discussion 里和杰里米讨论了，要用 fiber。
